tox.ini

```diff
@@ -63,7 +63,8 @@ max-line-length = 100
 [pydocstyle]
 # ignore the following
 #   - D413: no blank line after last section
-add-ignore = D413
+#   - D202: no blank line after docstring (handled by black, causes an issue if first line is '#pylint:disable')
```

Comment:
it can be solved by putting the disable above the function definition, but I am fine with this ignore :slightly_smiling_face:

Reply:
Moving the disable above the function won't have the same effect, because it would affect the full block and not only the single method (i.e. the disable affects all the methods defined after the disable).